go.sum

```diff
@@ -295,8 +295,8 @@ github.com/prometheus/procfs v0.0.0-20181005140218-185b4288413d/go.mod h1:c3At6R
 github.com/prometheus/procfs v0.0.0-20190117184657-bf6a532e95b1/go.mod h1:c3At6R/oaqEKCNdg8wHV1ftS6bRYblBhIjjI8uT2IGk=
 github.com/prometheus/procfs v0.0.2/go.mod h1:TjEm7ze935MbeOT/UhFTIMYKhuLP4wbCsTZCD3I8kEA=
 github.com/prometheus/procfs v0.0.8/go.mod h1:7Qr8sr6344vo1JqZ6HhLceV9o3AJ1Ff+GxbHq6oeK9A=
-github.com/pyalex/gopy v0.4.1-0.20220315220227-146bcd99e9f5 h1:+N63y8ub8/tUt4Z+U0WkoLpqX08biafsKfr1ZVe3Uus=
-github.com/pyalex/gopy v0.4.1-0.20220315220227-146bcd99e9f5/go.mod h1:ZO6vpitQ61NVoQP/2yOubPS6ET5pP3CAWCiMYn5eqCc=
+github.com/pyalex/gopy v0.4.1-0.20220328211859-c42fbc0e8c26 h1:HGVPQbIy3+ccH3rqX1K0ZE9mqDBubggTf2ZYaXTOaYE=
+github.com/pyalex/gopy v0.4.1-0.20220328211859-c42fbc0e8c26/go.mod h1:ZO6vpitQ61NVoQP/2yOubPS6ET5pP3CAWCiMYn5eqCc=
```

Comment:
Should we aim to get your changes upstreamed here? Or fork this repo into the feast-dev org?

Reply:
https://github.com/pyalex/gopy/commit/d705e6cd1d9bc95b5452fc60342315b40dd2c89b
This is the diff from gopy. Basically it uses not correct LD flags - same flags as used during building Python extension. Previously gopy used LD flags for compiling Python itself.

I'm happy to fork gopy in feast-dev for better visibility. @woop @adchia wdyt?

Reply:
If they flags were incorrect, would upstream the change, and eventually move to the main repo? And we fork into feast-dev to be unblocked?

Reply:
Will move gopy to feast-dev for now. Will prepare PRs to upstream a little later.